lib/src/main/java/uk/gov/di/ipv/cri/passport/library/service/ConfigurationService.java

```diff
@@ -200,4 +203,24 @@ public Certificate getClientJwtSigningCert(String clientId) throws CertificateEx
                                 + "/%s/sharedAttributesJwtSigningCert",
                         clientId));
     }
+
+    public List<String> getClientRedirectUrls(String clientId) {
+        Optional<String> redirectUrlStrings =
+                Optional.ofNullable(
+                        ssmProvider.get(
+                                String.format(
+                                        System.getenv("CREDENTIAL_ISSUERS_CONFIG_PARAM_PREFIX")
+                                                + "/%s/jwtAuthentication/validRedirectUrls",
+                                        clientId)));
```

Comment:
I know this is how we do this elsewhere, but I recently realised that the ssmProvider doesn't actually return null if a a param doesn't exist - it throws a specific error. I have a plan to fix it up where we do it. Happy for you to leave this as is and I'll do it all together, or you can change now if you'd rather?